ddtrace/globaltracer.go

```diff
@@ -1,14 +1,15 @@
 package ddtrace // import "github.com/signalfx/signalfx-go-tracing/ddtrace"
 
 import (
+	"sync"
+
 	"github.com/opentracing/opentracing-go"
 	"github.com/opentracing/opentracing-go/log"
-	"sync"
 )
 
 var (
 	mu           sync.RWMutex // guards globalTracer
-	globalTracer Tracer = &NoopTracer{}
+	globalTracer Tracer       = &NoopTracer{}
```

Comment:
formatting in strange here

Reply:
added blank line in between to force formatter not add so much blank space.

Reply:
basically the formatting behaviour is strange... this is not how `gofmt` would format it

Reply:
It does today 
<img width="1079" alt="image" src="https://user-images.githubusercontent.com/46186/114706289-643e2d00-9d46-11eb-8b35-063a97647b4a.png">

:)

Reply:
May be a bug in Go 1.16 but I really don't care as long as it comes from gofmt 😆

Reply:
I see. Sorry for wasting your time. Resolving